content/tracing/product_specs/metrics_namespace.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Traces metrics namespace
+kind: documentation
+---
+
+The trace metrics namespace is `trace.<name>.<metrics>{<tags>}` where
+
+* `<name>` is coming from the name of the integration ("redis", "pylons", etc.)
+* `<metrics>` is about the hits, errors or latency ("request.hits", etc.)
+* `<tags` the metrics are tagged by service & resource.
```

Comment:
Also `env` and any host tags when dealing with non-aggregated (ie. non histogram) metrics.

Reply:
I don't understand this.. :/